croutonjs/src/js/crouton-core.js

```diff
@@ -1124,6 +1127,31 @@ function getMasterFormatId(code, data) {
 	}
 }
 
+const venueType = {
+	IN_PERSON: 0,
+	VIRTUAL_TEMP: 1,
+	VIRTUAL: 2,
+	HYBRID: 3
+}
+
+function getVenueType(data) {
+	if (inArray(getMasterFormatId('HY', data), getFormats(data))
```

Comment:
I have been thinking about this. I think maybe the logic here should be slightly different than the logic for the selection in the root server UI. The use cases are a bit different for your average user trying to find a meeting and for your average service body admin.

In-Person - I think this should be inclusive of Hybrid meetings, because a Hybrid meeting is also technically an in-person meeting.

Virtual - I think this should be inclusive of Hybrid meetings. Maybe it should also be inclusive of VM,TC meetings, because Virtual (Temp) is technically still a virtual meeting.

I think your average person looking for a meeting would have trouble understanding the filtering as written.

Reply:
Actually, I'm not sure that I'm correct about this. We should discuss it. 

I do think @alanb2718 had a point about just lumping Virtual (Temp) in with Virtual, though.

Reply:
Agree, both in-person and virtual should include hybrid meetings for the reasons you stated.